server/src/repositories/IUsersRepositories.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from '@entities/User'
+
+interface IUsersRepositories {
+    findByEmail(email: string): Promise<User>
+    save(user: User): Promise<User>
+}
+
+export { IUsersRepositories }
```

Comment:
hear too

Reply:
Why are we speaking in English

Reply:
is good communication pattern and i'am practicing doing reviews in English